test/Webhook.spec.ts

```diff
@@ -152,13 +153,25 @@ describe('Webhooks', () => {
     };
   };
 
-  describe(
-    '.constructEvent',
+  describe('.constructEvent', () => {
     makeConstructEventTests(async (...args: any) => {
       const result = await stripe.webhooks.constructEvent(...args);
       return result;
-    })
-  );
+    })();
+    it('should provide helpful information when CryptoProviderOnlySupportsAsyncError is thrown', () => {
+      const header = stripe.webhooks.generateTestHeaderString({
+        payload: 'payload',
+        secret: 'secret',
+      });
+      expect(() => {
+        stripe.webhooks.constructEvent('payload', header, 'secret', 0, {
+          computeHMACSignature() {
+            throw new CryptoProviderOnlySupportsAsyncError('foobar');
+          },
+        });
+      }).to.throw(/Use `await constructEventAsync/);
```

Comment:
Does this actually check the message? Shouldn't the error be "/foobar\nUse `await constructEventAsync(...)` instead of `constructEvent(...)`/"?

Reply:
It runs the regex on the message. But I should include 'foobar' at least.